.github/files/e2e-tests/e2e-matrix.js

```diff
@@ -53,18 +77,19 @@ switch ( process.env.GITHUB_EVENT_NAME ) {
 		);
 
 		for ( const project of projects ) {
-			const packageJson = JSON.parse( fs.readFileSync( `${ project.path }/package.json`, 'utf8' ) );
+			// a test comment
```

Comment:
This can probably be removed.

Reply:
Indeed, removed with [836cb22](https://github.com/Automattic/jetpack/pull/28913/commits/836cb22e6b28eb0c31284c4ac8ae19a34d0d1c8b)

Reply:
Updated with [69fc3b4](https://github.com/Automattic/jetpack/pull/28913/commits/69fc3b462433405a8954093cf701b83a493b0304)